src/muya/lib/ui/quickInsert/config.js

```diff
@@ -19,6 +19,25 @@ import orderListIcon from '../../assets/icons/order_list.svg'
 // import sequenceIcon from '../../assets/icons/sequence.svg'
 // import mermaidIcon from '../../assets/icons/mermaid.svg'
 // import vegaIcon from '../../assets/icons/chart.svg'
+import { isOsx } from '../../config'
+
+const COMMAND_KEY = isOsx ? '⌘' : '⌃'
+
+// Command (or Cmd) ⌘
+// Shift ⇧
+// Option (or Alt) ⌥
+// Control (or Ctrl) ⌃
+// Caps Lock ⇪
+// Fn
+
+
+// Command (or Cmd) ⌘
+// Shift ⇧
+// Option (or Alt) ⌥
+// Control (or Ctrl) ⌃
+// Caps Lock ⇪
+// Fn
+
 
 export const quicInsertObj = {
   // 'diagram': [{
```

Comment:
I think we should enable the diagram fields.

Reply:
Not finished yet, I think there still some bugs when export HTML and PDF, I'll test this before enable these features, We'll release these features in the next release version.